package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "auth-be",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "test"
+  },
+  "scripts": {
+    "test": "jest --verbose --runInBand",
+    "test:watch": "npm run test -- --watch",
+    "lint": "eslint .",
+    "pretest": "npm run lint",
+    "start": "nodemon server.js",
```

```suggestion
    "start": "node server.js",
```
Looks like you have duplicate start scripts in this file. If you would like to have two, you can do something like the above where your start only runs the server once with node and a start:watch script that uses nodemon.